### PR TITLE
Update options.md

### PR DESCRIPTION
**Pull request by zzfima**
There is no connection between "Interface Segregation Principle (ISP)" and "Encapsulation". Linked text is about Encapsulation.

### CODE REVIEW

**Comment by guardrex** [comment]
Hello @zzfima ... Could you also update it on Line 241?

**Comment by guardrex** [comment]
@Rick-Anderson ... I think u should look. The wording of the whole line is strange.

**Comment by guardrex** [comment]
Thanks @zzfima!

Rick will be along in a bit to see if he wants to do anything else with the line. If not, he'll merge this shortly.

**Comment by zzfima** [comment]
@guardrex done

File: aspnetcore/fundamentals/configuration/options.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ By [Kirk Larkin](https://twitter.com/serpent5) and [Rick Anderson](https://twitt
 
 The options pattern uses classes to provide strongly typed access to groups of related settings. When [configuration settings](xref:fundamentals/configuration/index) are isolated by scenario into separate classes, the app adheres to two important software engineering principles:
 
-* The [Interface Segregation Principle (ISP) or Encapsulation](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#encapsulation): Scenarios (classes) that depend on configuration settings depend only on the configuration settings that they use.
+* The [Encapsulation](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#encapsulation): Scenarios (classes) that depend on configuration settings depend only on the configuration settings that they use.
 * [Separation of Concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns): Settings for different parts of the app aren't dependent or coupled to one another.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/configuration/options.md:21]:
```suggestion
* [Encapsulation](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#encapsulation):
  * Classes that depend on configuration settings depend only on the configuration settings that they use.
* [Separation of Concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns):
  * Settings for different parts of the app aren't dependent or coupled to one another.
```

File: aspnetcore/fundamentals/configuration/options.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ By [Kirk Larkin](https://twitter.com/serpent5) and [Rick Anderson](https://twitt
...
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/configuration/options.md:242]:
```suggestion
* [Encapsulation](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#encapsulation):
  * Classes that depend on configuration settings depend only on the configuration settings that they use.
* [Separation of Concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns):
  * Settings for different parts of the app aren't dependent or coupled to one another.
```

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**